pgjdbc/src/main/java/org/postgresql/translation/it.po

```diff
@@ -88,12 +97,12 @@ msgstr "Questo 
 
 #: org/postgresql/copy/PGCopyInputStream.java:110
 msgid "Read from copy failed."
-msgstr ""
+msgstr "Lettura da �copy� fallita."
 
 #: org/postgresql/copy/PGCopyOutputStream.java:71
 #, java-format
 msgid "Cannot write to copy a byte of value {0}"
-msgstr ""
+msgstr "Impossible scrivere in �copy� un byte il cui valore � {0}"
 
 #: org/postgresql/core/CommandCompleteParser.java:71
 #, fuzzy, java-format
```

Comment:
@the4thdoctor , could you please check this string? It looks like it is fuzzy and the translation might be off.